tests/TestCompose_fe-fe/referenceOutput/Dirichlet/precice-output/DirichletNodes-HeatDirichlet.final.vtk

```diff
@@ -63,15 +63,15 @@ VECTORS VertexNormals float
 SCALARS Temperature float
 LOOKUP_TABLE default
 4.5499999999999998e+00
-4.5799999666501376e+00
-4.6699999846989924e+00
-4.8200001397741170e+00
-5.0300001346214822e+00
-5.2999998400517230e+00
-5.6299999044882689e+00
-6.0200000931270825e+00
-6.4700000225582937e+00
-6.9800000432748952e+00
+4.5799999666542162e+00
+4.6699999847047362e+00
+4.8200001397763668e+00
+5.0300001346197218e+00
+5.2999998400495185e+00
+5.6299999044847784e+00
+6.0200000931221211e+00
+6.4700000225566550e+00
+6.9800000432747424e+00
 7.5499999999999998e+00
 
 VECTORS Flux float
```

Comment:
Why is there no change in the Flux for the final result? This is a bit puzzling, but might have a good reason I do not see now.

Reply:
It is even more puzzling that the flux in the `*.final.vtk` files in the [system test reference output](https://github.com/precice/systemtests/tree/develop/tests/TestCompose_fe-fe/referenceOutput/Dirichlet/precice-output) is zero in both X and Y directions. This is zero with the latest manual run of the tutorial too and hence there is no change.